game.py

```diff
@@ -382,6 +462,17 @@ def do_expose_event(self, event):
     def _destroy_cb(self, win, event):
         Gtk.main_quit()
 
+    def _new_dot_gameover(self, color):
+        ''' generate a dot of a color color '''
```

Comment:
There's a repetition of color at the end.

Reply:
@sourabhaa was this intended?

Reply:
sorry, I missed it earlier. Yes, the former refers to the noun color, latter refers to the argument `color`.